benchmarks/benchmarks/optimize_linprog.py

```diff
@@ -27,7 +62,45 @@ def klee_minty(D):
     A_ub = toeplitz(A_1, A1_)
     b_ub = np.array([5**(i + 1) for i in range(D)])
     c = -np.array([2**(D - i - 1) for i in range(D)])
-    return c, A_ub, b_ub
+    xf = np.zeros(D)
+    xf[-1] = 5**D
+    obj = c @ xf
+    return c, A_ub, b_ub, xf, obj
+
+
+class MagicSquare(Benchmark):
+
+    params = [
+        methods,
+        [(3, 1.7305505947214375), (4, 1.5485271031586025),
+         (5, 1.807494583582637), (6, 1.747266446858304)]
+    ]
+    if not slow:
+        params = [
+            methods,
+            [(3, 1.7305505947214375), (4, 1.5485271031586025)]
+        ]
```

Comment:
Would it be possible to avoid defining the (dimension, objetive) tuples twice for slow/not-slow cases?

Reply:
Sure. Done.